sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -255,6 +255,11 @@ public abstract partial class ClientOptions
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override string? ToString() { throw null; }
     }
+    public static partial class DelegatedTokenCredentialFactory
```

Comment:
I wonder if we should not call it simply TokenCredentialFactory. This would allow us to add more Create methods in the future even if they dont use delegates.

Reply:
My only concern with that is it could sound too much like the default option for creating a `TokenCredential`.

Reply:
It's a valid concern, but the type is in Azure.Core, and so it might be enough of a hint. Having said that, if we can come up with even a better name, it would be good. I just don't think we should name the type DelegatedTokenCredentialFactory.

Reply:
Renamed to `DelegatedTokenCredential`